src/Datadog.Trace.Tools.Runner/Crank/Importer.cs

```diff
@@ -111,7 +111,7 @@ public static int Process(string jsonFilePath)
 
                         span.SetTag(TestTags.Name, jobItem.Key);
                         span.SetTag(TestTags.Type, TestTags.TypeBenchmark);
-                        span.SetTag(TestTags.Suite, fileName);
+                        span.SetTag(TestTags.Suite, $"Crank.{fileName}");
```

Comment:
I think you should do the same change in line [146](https://github.com/DataDog/dd-trace-dotnet/blob/3cfd4cea399ef5a085960bf196f9fa33f418d0b6/src/Datadog.Trace.Tools.Runner/Crank/Importer.cs#L146).

Reply:
What do you think of the new changes?

Reply:
Whoops, I didn't realize that I was overriding the values being set by the properties. I fixed that in the latest commits, so I think it should be good to merge now